### PR TITLE
[PRTL-2622] empty row

### PR DESCRIPTION
**Pull request by samrichca**
## Ticket Number

PRTL-2622

## Environment to be used in testing

- [ ] `prod`
- [x] `dev-oicr`
- [ ] `qa` (which version?)

## Description of Changes

Adds a permanent new row to the manual range table for continuous binning.

Removes functionality for adding new rows when the last row is saved or deleted.

### CODE REVIEW

File: src/packages/@ncigdc/components/Modals/ContinuousBinning/ContinuousCustomBinsModal.js [hunk 1]
```diff
@@ -39,12 +31,14 @@ class ContinuousCustomBinsModal extends Component {
...
-    rangeRows: defaultRangeRow,
+    rangeRows: [],
   };
 
   componentDidMount = () => {
     const {
-      binData, continuousBinType, rangeRows,
+      binData,
+      continuousBinType,
+      rangeRows,
```
Comment (caravinci) [inline src/packages/@ncigdc/components/Modals/ContinuousBinning/ContinuousCustomBinsModal.js]: 🙌

Comment (samrichca) [reply src/packages/@ncigdc/components/Modals/ContinuousBinning/ContinuousCustomBinsModal.js]: haiku coding pattern

File: src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js [hunk 2]
```diff
@@ -849,7 +849,12 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
               <BarChart
                 data={chartData}
                 height={CHART_HEIGHT}
-                margin={{ top: 20, right: 50, bottom: 50, left: 55 }}
+                margin={{
+ top: 20,
+right: 50,
+bottom: 50,
+left: 55 
+}}
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: what's up with the indent here?

Comment (samrichca) [reply src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: very good question, I tried to copy-paste the original file from cdave branch to correct/revert the changes to this file but it didn't work.

need to fix my vscode next week

Comment (samrichca) [reply src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: fixed in a merge conflict in another PR!

Comment (caravinci) [reply src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: thanks! also fixed it in mine.

**Pull request merged**